templates/chartmuseum/chartmuseum-cm.yaml

```diff
@@ -55,7 +55,9 @@ data:
 {{- else if eq $storageType "gcs" }}
   STORAGE: "google"
   STORAGE_GOOGLE_BUCKET: {{ $storage.gcs.bucket }}
+  {{- if not .Values.persistence.imageChartStorage.gcs.useWorkloadIdentity }}
```

Comment:
Could you please justify why we need the configuration of: 
* `.Values.persistence.imageChartStorage.gcs.useWorkloadIdentity`

Same questions for subsequent `useWorkloadIdentity` config. If there is not a strong justification of `useWorkloadIdentity`. I would suggest delete it.

Reply:
Hi,
as stated here for exemple: [https://gcloud.readthedocs.io/en/latest/google-cloud-auth.html#credential-discovery-precedence](https://gcloud.readthedocs.io/en/latest/google-cloud-auth.html#credential-discovery-precedence) , if the "GOOGLE_APPLICATION_CREDENTIALS" variable will take precedence and the application will not be able to use the current kubernetes service account to authenticate.
When using workloadidentity, we need to remove any other gcloud sdk authentication mechanism.